src/cascadia/WindowsTerminal/WindowEmperor.cpp

```diff
@@ -683,7 +686,7 @@ void WindowEmperor::_createMessageWindow(const wchar_t* className)
         /* Y            */ 0,
         /* nWidth       */ 0,
         /* nHeight      */ 0,
-        /* hWndParent   */ HWND_MESSAGE,
+        /* hWndParent   */ nullptr,

```

Comment:
can somebody display our window now?

Reply:
Yes. Maybe I can hook `WM_WINDOWPOSCHANGING` and clear the bits? 🤔